### ISSUE TITLE
Add markdown code tabs extension

### ISSUE THREAD

**Issue by froschdesign** [zendframework/zf-mkdoc-theme#68]
# Problem

At the moment the documentation often only describes the standalone use of a component. Concrete examples are missing how the end user can use the component in his application.

# Suggestion

Besides [longer examples](https://github.com/zendframework/documentation/issues/8) of integrations which needs extra pages in the documentation, we should also add an option for simpler code blocks.  This can be done with a tabbed navigation; short "tabs".

## Examples

* mvc-based and expressive-based applications (like controller-action and handler)
* different renderer (like Twig, Plates, zend-view)
* usage standalone or embedded in a class (like `Zend\Db\Sql\Sql` as standalone or in a `TableGateway` class)
* array notation vs. usage with methods in forms, validators, filters, etc.

# Implementation

## Markdown extension

The extension "[Code Tabs](https://yacir.github.io/markdown-fenced-code-tabs/)" for Python-Markdown allows us to create code examples for different uses and application integrations. This extension comes with ready-to-use template for the CSS framework Bootstrap.

We must extend the configuration for MkDocs: (`mkdocs.yml`)

```yml
markdown_extensions:
    - markdown_fenced_code_tabs:
        template: bootstrap3
    - markdown.extensions.codehilite:
        use_pygments: False
```

### Example

<pre>
```php fct_label="Standalone Use"
$sql    = new Zend\Db\Sql\Sql($adapter, 'foo');
$select = $sql->select();
$select->where(['id' => 2]);
```
</pre>

<pre>
```php fct_label="TableGateway Use"
$select = $this->getSql()->select();
$select->where(['id' => 2]);
```
</pre>

(GitHub ignores the extra markup `fct_label` in the preview.)

## Preview

![tabs](https://user-images.githubusercontent.com/103362/48628453-81287780-e9b7-11e8-92c9-b54d64c2c34d.png)

Documentation: https://yacir.github.io/markdown-fenced-code-tabs/

## Bootstrap component

https://getbootstrap.com/docs/3.3/components/#nav-tabs

**Comment by weierophinney** [comment]
Addressed with #70; documentation can now start using nav tabs for code examples. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by froschdesign** [comment]
@weierophinney 
I will create a documentation guide that covers features like this.
